### PR TITLE
jax.numpy.choose

### PR DESCRIPTION
**Pull request by amoungui**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Review by AryanSharma21 — COMMENTED** [review]
hey @amoungui is there any open issue that you might have created for this Pull request ?

**Comment by amoungui** [comment]
Yes, I think I'm a little mixed up with my internet connection, I thought
that the creation of the issue was not done when my internet connection
broke, so I did a once again proceed.

Le mer. 16 août 2023 à 19:29, Aryan sharma ***@***.***> a
écrit :

**Comment by Daniel4078** [comment]
Please don't forget to resolve the merge conflic @amoungui

**Comment by amoungui** [comment]
Yes, I am working on it.

Le mer. 23 août 2023 à 00:50, Daniel4078 ***@***.***> a
écrit :

File: ivy/functional/frontends/jax/numpy/indexing.py [hunk 1]
```diff
@@ -182,6 +188,8 @@ def __len__(self) -> int:
 
 
 class RClass(_AxisConcat):
+    """RClass represents"""
```
Comment (Daniel4078) [inline ivy/functional/frontends/jax/numpy/indexing.py:191]: similarly, why is this here?

File: ivy/functional/frontends/jax/numpy/indexing.py [hunk 2]
```diff
@@ -101,6 +101,12 @@ def indices(dimensions, dtype=int, sparse=False):
         return ivy.stack(grid, axis=0).astype(dtype)
 
 
+@to_ivy_arrays_and_back
+def choose(arr, choices, out=None, mode="raise"):
+    """jax.numpy.choose(arr, choices, out=None, mode='raise')"""
```
Comment (Daniel4078) [inline ivy/functional/frontends/jax/numpy/indexing.py:106]: Why there is this comment here?

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
found a little problem and good job getting all the relevent tests passing

File: ivy/functional/frontends/jax/numpy/indexing.py (hunk 2 above)
Comment (amoungui) [reply ivy/functional/frontends/jax/numpy/indexing.py:106]: because I wanted to resolve lint warnings. I will move that, I think is better

**Comment by Daniel4078** [comment]
ivy-gardener
